client/web/reporter/src/mixins/report.js

```diff
@@ -90,5 +90,23 @@ export default {
           this.processingDelete = false
         })
     },
+
+    handleClone (report) {
+      this.processing = true
+      const { handle, meta, sources, blocks, scenarios, labels } = report
+      meta.name = `${meta.name} (${this.$t('general:cloneSuffix')})`
+      this.processingSave = true
```

Comment:
Make sure the processing is on the correct button.

A couple of rules:
1. Disable other buttons that trigger API actions if something is processing
2. Make sure that only the button that was pressed is processing